cloud_infra_center/ocp_upi/common.yaml

```diff
@@ -0,0 +1,45 @@
+# =================================================================
+# Copyright 2021 https://github.com/openshift/installer
```

Comment:
You have a URL going out to another repo here after the copyright. "https://github.com/openshift/installer"

Reply:
Yes, because this yaml file was copied from OpenShift repo, I was told to keep the same copyright for legal reason.

Reply:
That makes sense, you are correct that we need to honor other parties copyrights , good thing is its the same License, but I still want to check with Legal to see if this is enough to have a link or do we need more.

Reply:
Thanks, sounds good~